File: 1975-minimum-distance-to-the-target-element/1975-minimum-distance-to-the-target-element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        mi = 10001
        for i in range(len(nums)):
            if nums[i] == target:
                d = abs(i - start)
                if d < mi:
                    mi = d
        return mi

File: 1975-minimum-distance-to-the-target-element/1975-minimum-distance-to-the-target-element.java
class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        int min = 10001;
        for(int i = 0; i < nums.length; i++){
            if(nums[i] == target){
                int d = Math.abs(i - start);
                if(d < min){
                    min = d;
                }
            }
        }
        return min;
    }
}